plugins/loading/rollup.config.js

```diff
@@ -1,6 +1,6 @@
 import replace from 'rollup-plugin-replace'
 import uglify from 'rollup-plugin-uglify'
```

Comment:
Could update to the latest uglify, making the minify import unnecessary.

https://www.npmjs.com/package/rollup-plugin-uglify

It requires a named import.

`import { uglify } from "rollup-plugin-uglify";`

Reply:
@ShMcK After I upgraded, the plugin didn't work.

Reply:
Here's an example: https://github.com/rematch/rematch/blob/a77f811c8b65c864cba7caf102cb3c6173b255f3/rollup.config.js#L4

Reply:
@ShMcK Updated, please check the new code.